start_jupyter.py

```diff
@@ -120,31 +109,35 @@ def parse_arguments():
     parser.add_argument('--ram',
                         default=8000, type=int,
                         help='MB of RAM to request')
-    parser.add_argument('--conda_path',
-                        default='<INFER>',
-                        help="For non-singularity environments, path to conda binary to use."
-                             "Default is to infer this from running 'which conda'.")
     parser.add_argument('--gpu',
                         action='store_true', default=False,
                         help='Request to run on a GPU partition. Limits runtime to 2 hours.')
     parser.add_argument('--env',
-                        default='nt_singularity',
-                        help='Environment to activate; defaults to "nt_singularity" '
+                        default='singularity',
+                        choices=['singularity', 'cvmfs'],
+                        help='Environment to activate; defaults to "singularity" '
                              'to load XENONnT singularity container. '
-                             'Other arguments are passed to "conda activate" '
-                             "(and don't load a container).")
-    parser.add_argument('--container',
-                        default='xenonnt-development.simg',
-                        help='Singularity container to load'
+                             'Passing "cvmfs" will use the conda environment installed in cvmfs, '
+                             'using the container argument to determine which env exactly ')
```

Comment:
should it not be "using the tag argument"?

Reply:
Yes it should, thanks!